http/refresh/navigate.window.js

```diff
@@ -0,0 +1,23 @@
+async_test(t => {
+  const frame = document.createElement("iframe");
+  frame.src = "resources/refresh.py"
+  frame.onload = t.step_func(() => {
+    // Could be better by verifying that resources/refresh.py loads too
+    if(frame.contentWindow.location.href === (new URL("resources/refreshed.txt?ÿ", self.location)).href) {
```

Comment:
Is this supposed to be testing the decoding? If so, a comment or test name string would be good. Also it might be good to verify that the bytes actually make it back through Python correctly. Although I think given Python's very naive Unicode handling that's probably true.